packages/flutter/lib/src/widgets/scroll_physics.dart

```diff
@@ -34,7 +39,14 @@ class ScrollPhysics {
   /// If a subclass of [ScrollPhysics] does not override a method, that subclass
   /// will inherit an implementation from this base class that defers to
   /// [parent]. This mechanism lets you assemble novel combinations of
-  /// [ScrollPhysics] subclasses at runtime.
+  /// [ScrollPhysics] subclasses at runtime. For example:
+  ///
+  /// ```dart
+  /// BouncingScrollPhysics(parent: AlwaysScrollPhysics())
```

Comment:
All doc sample codes need to be compilable. This line's missing a ;

Reply:
The sample code analyzer doesn't expect the trailing `;` if it's a constructor iirc.